src/basic/time-util.c

```diff
@@ -1381,73 +1381,66 @@ int get_timezones(char ***ret) {
         return 0;
 }
 
-bool timezone_is_valid(const char *name, int log_level) {
+int verify_timezone(const char *name, int log_level) {
```

Comment:
verify or validate?

Reply:
I called it "verify" because there's already "fd_verify_regular" used in the body. I like "verify" because it makes me think that the function will fail if the conditions are not met.